src/assets/constants.ts

```diff
@@ -26,3 +26,9 @@ export const questions = [
   "If you had enough money to do whatever you want to, what would you be doing?",
   "If you could travel anywhere for a year, where would you go?"
 ];
+
+// step 1: make an array with 1 or more questions
+const bobsQuestions = ["What is your favorite color?", "What is your favorite food?", "What is your favorite movie?"];
```

Comment:
I noticed you named this bobsQuestions. Can we make it more specific? like:
```suggestion
const basicQuestions = ["What is your favorite color?", "What is your favorite food?", "What is your favorite movie?"];
```